src/lib/style-properties.html

```diff
@@ -179,6 +178,20 @@
         rule.cssText = output;
       },
 
+      // Apply keyframe transformations to the cssText of a given rule. The
+      // keyframeTransforms object is a map of keyframe names to transformer
+      // functions which take in cssText and spit out transformed cssText.
+      applyKeyframeTransforms: function(rule, keyframeTransforms) {
+        var output = rule.cssText;
+        if (this.rx.ANIMATION_MATCH.test(output)) {

```

Comment:
In fact since we don't expect the of animation names to change, can we memo-ize them and then only run the relevant transform functions?